src/envo/envo-edit.owl

```diff
@@ -38994,10 +39007,17 @@ SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03500004> <http://purl.obolibrar
 # Class: <http://purl.obolibrary.org/obo/ENVO_03500005> (anthropogenic litter)
 
 AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03500005> "An anthropogenic environmental material consisting of any object no longer wanted and discarded outside of approved trash receptacles.")
```

Comment:
>An anthropogenic environmental material consisting of any object no longer wanted and discarded outside of approved trash receptacles.


Not a fan of "trash", and litter need not be anthropogenically sourced. Further, in this case, litter isn't really a subclass of environmental material, more a role realised by it. Thus I would add: 

Under BFO:role:
> litter role =def. A role which 1) inheres in a material entity 2) is realised when that entity is discarded outside waste management systems by humans or their technologies.
 
and then, under BFO:material entity:
> anthropogenic litter =def. a material entity which has been discarded outside waste management systems by humans or their technologies.

Axioms would follow, and need a "littering process" to also be defined in a similar way. 

I also wonder if we need some sort of size cut-off. Very large things that are discarded are typically considered "debris" but this is quite loosely defined too.

Reply:
> I also wonder if we need some sort of size cut-off. Very large things that are discarded are typically considered "debris" but this is quite loosely defined too.

For the time being, we could add a comment noting this - however, as there is not clearly defined cut-off, I wouldn't add it to the definition.

Reply:
> I also wonder if we need some sort of size cut-off. Very large things that are discarded are typically considered "debris" but this is quite loosely defined too.

For the time being, we could add a comment noting this - however, as there is not clearly defined cut-off, I wouldn't add it to the definition.